### PR TITLE
`KIE Tools` --> `Apache KIE Tools`

### PR DESCRIPTION
**Pull request by tiagobento**
Updating references and names.

### CODE REVIEW

File: .github/actions/checkout-pr/action.yml
Comment (rodrigonull) [inline .github/actions/checkout-pr/action.yml:1]: Should we also update line 38? I noticed that the existing forks are user/kie-tools but the new one are going to be user/incubator-kie-tools.
My suggestion is to change this line to:
`git remote add $user https://github.com/$user/${{ github.event.pull_request.head.repo.name }}.git`

**Review by rodrigonull — CHANGES_REQUESTED** [review]
Can you also update the file `.ci/jenkins/shared-scripts/pipelineVars.groovy` and replace `kit-tools` -> `incubator-kie-tools`?

File: .github/actions/checkout-pr/action.yml
Comment (tiagobento) [reply .github/actions/checkout-pr/action.yml:1]: Great catch. Changed!

**Comment by tiagobento** [comment]
@rodrigonull Changes applied here: [16f0934](https://github.com/apache/incubator-kie-tools/pull/2082/commits/16f0934f9faf6f9a06fea6a15ca9d2341239c20d)

**Review by rodrigonull — APPROVED** [review]

**Pull request merged**